pkg/framework/dataplane/loopback.go

```diff
@@ -108,7 +115,18 @@ func (ldp *loopbackDataPlane) verifyOnPort(port uint32, pkts [][]byte) bool {
 			return result
 		// The default case is only for passing CI and will be updated/removed once implemetation completes
 		default:
```

Comment:
In this case it always runs the default branch and the timer won't work which makes it inconsistent with the direct mode

Reply:
We'll keep as is in this patch and address the issue after more discussions

Reply:
Keeping as is in this patch, to be fixed after more discussions.